### DISCUSSION TITLE
Notification Channel always goes to general

### DISCUSSION THREAD

**Discussion by pjones19832023** [thudugala/Plugin.LocalNotification#456] [General]
When I try to send a notification it always goes to the General notification channel even though I have created the correct channel and it shows in Android. it even creates new General channels when I disable the general channel. This occurs on android but hasn't been tested on ios. 

            notificationRequest.Android.ChannelId = "TestChannel";

having checked the plugin code below, it only seems to use the default channel name

 public static string DefaultChannelName => "General";

 /// <summary>
 /// Default Group Id
 /// </summary>
 public static string DefaultGroupId => "Plugin.LocalNotification.Group";

 /// <summary>
 /// Default Group Name
 /// </summary>
 public static string DefaultGroupName => "GeneralGroup";

 /// <summary>
 /// Setting this flag will make it so the notification is automatically canceled when the user clicks it in the panel.
 /// Default is true
 /// </summary>
 public bool AutoCancel { get; set; } = true;

 /// <summary>
 /// Sets or gets, The id of the channel. Must be unique per package. The value may be truncated if it is too lon
 /// Use this to target the Notification Channel.
 /// </summary>
 public string ChannelId { get; set; } = DefaultChannelId;

**Comment by thudugala** [comment]
@pjones19832023 can you please attach a sample project?
